bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/.gitignore

```diff
@@ -0,0 +1 @@
+/RoboMapViewer.java
```

Comment:
Why are you adding a gitignore file here?

Reply:
I discovered that classes in the test folder are not in the regular jar file.
Hence to build the standalone viewer, https://community.openhab.org/t/xiaomi-vacuum-map-viewer-to-find-coordinates-for-zone-cleaning/103500 , I needed to move the file to the regular/main folder.

to prevent it from being uploaded I clicked ignore. seems that generates the .gitignore file. 
I'll remove the .gitignore file as it is no longer needed